### ISSUE TITLE
Outlier Detection & Handling

### ISSUE THREAD

**Issue by frederikpahde** [Fred93/BADS#6]

**Comment by Selina91** [comment]
http://link.springer.com/chapter/10.1007%2F978-3-642-01082-8_9#page-1
Neural networks and outliers

**Comment by Selina91** [comment]
Outlier robust (list is not complete):
-   tree, random forest, clustering, k-nearest neighbors
-   controlling outliers by using the robust Mahalanobis distance based on principal component analysis (PCA)
-   robust regularized regression by using a weight through the Mahalanobis distance in principal component space
-   By using the Gaussian kernel function in regularized regression
-   penalized least trimmed squares criterion

Not robust (list is not complete):
-   least square loss function
-   regularized regression modeling with a lasso penalty
-   mean squared error as error measure in general
